chainercv/transforms/image/resize.py

```diff
@@ -1,13 +1,17 @@
+import numpy
 import warnings
 
 try:
     import cv2
 
     def _resize(img, size):
-        return cv2.resize(img, dsize=size)
+        img = cv2.resize(img, dsize=size)
+        if len(img.shape) == 3:
```

Comment:
Could you add an argument that explains this condition?
I thought it was not intuitive that `cv2` changed shape of img from `H, W, 1` to `H, W`.

So, add something like
"# When channel size of the image is one, the resized image returned by cv2 is two dimensional array.